Objects/Spawners/DwarvesManager/DwarvesManager.gd

```diff
@@ -2,20 +2,18 @@ extends SpawnerSystem
 
 export(float) var dwarf_max_hp : float = 2.0
 export(float) var dwarf_damage : float = 1.0
-export(float) var boss_max_hp : float = 5.0
-export(float) var boss_damage : float = 2.0
 export(bool) var spawn : bool = true
 
 var Dwarf = load("res://Objects/Dwarves/Dwarf/Dwarf.tscn")
 var Boss = load("res://Objects/Dwarves/Boss/Boss.tscn")
 
-# dwarf data on level 1
 onready var default_dwarf_hp = dwarf_max_hp
 onready var default_dwarf_damage = dwarf_damage
+onready var WPHK = 0.0305
```

Comment:
This name means nothing. I recommend to write something like this: `hp_increase_ratio` and: `damage_increase_ratio`.

Reply:
Yes, you are right